src/data.js

```diff
@@ -197,8 +231,19 @@ const compareTagToCommit = pMemoize(
 
 const getTSDefs = pMemoize(
   async (version) => {
-    const file = await fetch(`https://unpkg.com/electron@${version}/electron.d.ts`);
-    return await file.text();
+    try {
+      const file = await fetch(`https://unpkg.com/electron@${version}/electron.d.ts`);
+      if (!file.ok) {
+        console.error(
+          `Failed to fetch TypeScript definitions for ${version}: ${file.status} ${file.statusText}`,
+        );
+        return '// TypeScript definitions could not be loaded';
+      }
+      return await file.text();
+    } catch (error) {
+      console.error(`Error fetching TypeScript definitions for ${version}:`, error);
+      return '// TypeScript definitions could not be loaded';
+    }
```

Comment:
Returning `'// TypeScript definitions could not be loaded'` on error is not correct here, and because this `pMemoize` caches for 24 hours this would result in a bad value being cached for 24 hours due to a transient network error, which is very undesirable behavior.